src/Modules/Backend/assets/Backend/webpack/js/Components/InitToasts.js

```diff
@@ -0,0 +1,15 @@
+export class InitBsToasts {
+  constructor () {
+    this.initToasts()
+  }
+
+  // init toasts present on page load
+  initToasts () {
+    const messageNodes = document.querySelector('[data-messaging-wrapper]').querySelectorAll('.toast')
```

Comment:
Wouldn't be more future proof to not use the class `.toast` to select the elements? But a data-attribute instead?

Reply:
True, updated